pkg/operation/hybridbotanist/controlplane.go

```diff
@@ -145,6 +157,30 @@ func (b *HybridBotanist) DeployETCD() error {
 		return err
 	}
 
+	// Apply etcd main
```

Comment:
What's this? Why is this commented out?

Reply:
yeah not needed. I was about to push cleanup changes.